maintainers/maintainer-list.nix

```diff
@@ -13167,6 +13167,13 @@
     githubId = 191622;
     name = "Denys Pavlov";
   };
+  meator = {
+    email = "meator.dev@gmail.com";
+    github = "meator";
+    githubId = 67633081;
+    name = "meator";
+    keys = [ { fingerprint = "7B0F 58A5 E0F1 A2EA 1157  8A73 1A14 CB34 64CB E5BF"; } ];
```

Comment:
please sign your commit with that key.

Reply:
Ah, I forgot to enable gpgsign in the clone.